serverless/aws/functions/generic.py

```diff
@@ -46,6 +46,7 @@ def __init__(
     ):
         super().__init__()
         self._service = service
+        self.layers = []
```

Comment:
The layers argument is now unused, and there is code below doing `layers.append(...)` which now achieves nothing.

Reply:
It is still used on line 266.

Reply:
Line 266 is referring to `self.layers` but I am talking about the `layers` argument/variable in this function that gets altered and never used.

Reply:
Ah ok, got you.

Reply:
There is still the section doing `layers.append(...)`

https://github.com/epsylabs/serverless-builder/blob/984e0102da585a94b552357e4130f99384c0adcc/serverless/aws/functions/generic.py#L69-L78

Reply:
Missed that, fixed.